app/views/thredded/messageboards/index.html.erb

```diff
@@ -9,8 +9,11 @@
         <% @groups.each do |group| %>
           <% if group.name.present? %>
             <h3 class="thredded--messageboards-group--title">
-              <%= link_to group.name,
-                          show_messageboard_group_path(group.id) %>
+              <%= link_to_if Thredded.show_messageboard_group_page,
+                            group.name,
+                            show_messageboard_group_path(group.id) do 
+                              group.name
+                            end%>
```

Comment:
You can remove the `do...end` part, since it's the same as the second argument (`group.name`)

Reply:
Corrected.